documentation/asciidoc/topics/attributes/product-attributes.adoc

```diff
@@ -61,13 +60,6 @@
 :fullbrandname: Red Hat Data Grid
 :wildflybrandname: EAP
 
-:infinispanversion: 8.0
-:infinispancomponentversion: 10.0.0-SNAPSHOT
-
-:brandcli: cli
-:moduleprefix: jdg
-:infinispanslot: jdg-8.0
-
 :download_url: https://access.redhat.com/downloads
 :javadocroot: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/7.3/api
 :configdocroot: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/7.3/Configuration/
```

Comment:
`:runtime_readme:` (a couple of lines below) can be removed. 
it is pointing to the new server.